docs/sbc_sharc.md

```diff
@@ -0,0 +1,355 @@
+# nf-core/configs: Sheffield Bioinformatics Core Facility ShARC Configuration
+
+The following nf-core pipelines have been successfully configured for use on the the [University of Sheffield ShARC cluster](https://docs.hpc.shef.ac.uk/en/latest/index.html):
+
+- [nf-co.re/atacseq](https://nf-co.re/atacseq)
+- [nf-co.re/chipseq](https://nf-co.re/chipseq)
+- [nf-co.re/rnaseq](https://nf-co.re/rnaseq)
+- [nf-co.re/sarek](https://nf-co.re/sarek)
+
+When using [`sbc_sharc.config`](../conf/sbc_sharc.config) with the pipelines listed above, the appropriate configuration file from the list below will be loaded automatically:
+
+- [atacseq sbc_sharc.config](../conf/pipeline/atacseq/sbc_sharc.config)
+- [chipseq sbc_sharc.config](../conf/pipeline/chipseq/sbc_sharc.config)
+- [rnaseq sbc_sharc.config](../conf/pipeline/rnaseq/sbc_sharc.config)
+- [sarek sbc_sharc.config](../conf/pipeline/sarek/sbc_sharc.config)
+
+The [`sbc_sharc.config`](../conf/sbc_sharc.config) configuration file might work with other nf-core pipelines as it stands but we cannot guarantee they will run without issue. We will be continuing to create, test and optimise configurations for new pipelines in the future.
```

Comment:
I find this a bit confusing the start with the pipelines it's configured for, the base image should work with most and I owuld normally start with that and with a (generic) command  right at the beginning showing how to load the base config (i.e. the next section should go first)